package.json

```diff
@@ -12,6 +12,7 @@
     "react-native-button": "1.7.1",
     "react-native-drawer": "2.3.0",
     "react-native-keyboard-spacer": "0.3.0",
+    "react-native-svg": "4.3.3",
```

Comment:
Does anything need to be added to NOTICE.txt for this? Even if we don't do it now, maybe make a ticket so that we don't forget it